vyper/semantics/analysis/module.py

```diff
@@ -308,13 +309,15 @@ def visit_InterfaceDef(self, node):
         obj = InterfaceT.from_ast(node)
         try:
             self.namespace[node.name] = obj
+            self.namespace.add_to_module_custom_types(node.name, obj)
```

Comment:
todo: remove the add_to_module_custom_types stuff, we can approach it a little differently

Reply:
done in 2f057d052d7cfa2ad3ad8b6cde7cefeb53015ad8